code-studio/src/js/components/share_dialog.jsx

```diff
@@ -1,5 +1,5 @@
 var ShareDialogBody = require('./share_dialog_body.jsx');
-var Dialog = require('./dialog.jsx');
+var Dialog = require('@cdo/apps/templates/Dialog.jsx');
```

Comment:
Is this the right directionality? (code-studio requiring files from apps vs. having it live in code-studio, and apps pulls it in from there?)